services/order.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order | None:
+
+    with (transaction.atomic()):
+        try:
+            user = get_user_model().objects.get(username=username)
+        except ObjectDoesNotExist as e:
+            raise ValidationError(str(e))
```

Comment:
Raising a `ValidationError` with the caught exception message might not be the best approach. Consider providing a more user-friendly error message to indicate that the user could not be found.